Makefile.am

```diff
@@ -20,6 +20,9 @@ nodist_systemdunit_DATA =
 
 TESTS = $(NULL)
 
+# dependency on "npm install"
+NPM = package-lock.json
```

Comment:
As discussed on IRC, I don't like this name for the variable, because this file is not npm.  Pick something that makes it sound more explicitly like an output/result/stamp of npm, please.

Reply:
How about `NPM_INSTALL` or `NODE_MODULES`?